tutorials/tutorial-7-mdpa.py

```diff
@@ -0,0 +1,42 @@
+import pygem as pg
+from pygem.mdpahandler import MdpaHandler
+import vedo
+import meshio
+
+ffd = pg.FFD()
+ffd.read_parameters('../tests/test_datasets/parameters_test_ffd_pipe_unv_C0.prm')
+
+%cat ../tests/test_datasets/parameters_test_ffd_pipe_unv_C0.prm
+
+mdpa_handler = MdpaHandler()
+mesh_points = mdpa_handler.parse('../tests/test_datasets/test_pipe.mdpa')
+
+UndeformedMDPA = meshio.read('../tests/test_datasets/test_pipe.mdpa')
+meshio.write('test_pipe_undeformed_mdpa.vtk', UndeformedMDPA)
+UndeformedVTK = vedo.load('test_pipe_undeformed_mdpa.vtk')
+vedo.settings.embedWindow(backend='k3d', verbose=True)
+vedo.show(UndeformedVTK, viewup="z", resetcam=True)
+
+new_mesh_points = ffd(mesh_points)
+
+DeformedMDPA = meshio.read('test_pipe_mod_C0.mdpa')
+meshio.write('test_pipe_deformed_mdpa.vtk', DeformedMDPA)
+DeformedVTK = vedo.load('test_pipe_deformed_mdpa.vtk')
+vedo.settings.embedWindow(backend='k3d', verbose=True)
+vedo.show(DeformedVTK, viewup="z", resetcam=True)
+
+%cat ../tests/test_datasets/parameters_test_ffd_pipe_unv_C1.prm
```

Comment:
same here

Reply:
I think it is good here as the mdpa can therefore be visualized

Reply:
Yes, for the external packages. I was referring to the `cat` line. Thanks